scripts/go-to-slide.js

```diff
@@ -7,8 +7,10 @@ H5P.CoursePresentation.GoToSlide = (function ($) {
    * @class
    * @param {Number} slideNum
    * @param {CoursePresentation} cp
+   * @param {String} goToSlideNextPrevious
+   * @param {Number} index
    */
-  function GoToSlide(title, slideNum, invisible, cp) {
+  function GoToSlide(title, slideNum, invisible, cp, goToSlideNextPrevious, index) {
```

Comment:
hehe, so I think it is zero when the constructor is run, but it should be correct when the go function is run. I haven't checked though, but I know you're on slide 0 when everything is constructed so that variable should be 0. When the go function is run you're on the page where the button is so cp.currentSlideIndex should be the index of the slide where the button is placed.